### DISCUSSION TITLE
git merge conflict -> plan file doesn't match changes table

### DISCUSSION THREAD

**Discussion by jcmrva** [theory/sqitch#631] [Q&A] [answered]
In brief:
- My coworker and I start from the same point.
- We both make different local changes & push them.
- The changes to the `sqitch.plan` file result in a merge conflict as expected.
- Running `sqitch deploy` after pulling the merged plan file results in this situation:
> Cannot find change 76f2a... (name) in sqitch\sqitch.plan

Plan file
|---|
| A |
| B |
| C |

Local DB changes table
|---|
| A |
| C |

The fix was easy enough: revert local DB to the last matching change & deploy again.

I'm wondering if there's a way to make it automatic - we're hoping to include sqitch deployments in a build script.

**Comment by theory** [comment, marked as answer]
This is exactly what [`sqitch-rebase --modified`](http://sqitch.org/docs/manual/sqitch-rebase/) was designed to handle. Use it to revert to the last change in sync with the database, then re-deploy.
